.github/workflows/dotnet-build.yml

```
@@ -28,6 +28,7 @@ jobs:
   analyze:
     name: Analyze
     runs-on: windows-latest
+    if: github.repository == "3PillarGlobal-Czechia/interview-app-ui"
```

I believe we are in backend repository right now? 🤔 
```suggestion
    if: github.repository == "3PillarGlobal-Czechia/interview-app-api"
```